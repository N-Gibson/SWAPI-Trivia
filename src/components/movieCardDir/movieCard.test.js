import React from 'react';
import { shallow } from 'enzyme';
import MovieCard from './movieCard';

describe('Character Container', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<MovieCard />)
  })

  it.skip('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();    
  })
})