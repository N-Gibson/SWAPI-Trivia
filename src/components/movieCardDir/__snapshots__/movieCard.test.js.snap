// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character Container should match snapshot 1`] = `undefined`;
